CHANGELOG.md

```diff
@@ -8,7 +8,8 @@
 ### Features Added
 - Signing support for BlobSidecar and BlindedBlobSidecar in Deneb fork.
 - Add `--azure-response-timeout` to allow request response timeout to be configurable, the field `timeout` is also accepted in the Azure metadata file. [#888](https://github.com/Consensys/web3signer/pull/888)
-- Bulk load Ethereum v3 wallet files in eth1 mode. 
+- Bulk load Ethereum v3 wallet files in eth1 mode.
+- Add network configuration for Holesky testnet
```

Comment:
Do we want to add the link of the PR in the changelog? I usually do add them and personally like to have them there